fido-key-manager/src/main.rs

```diff
@@ -774,5 +777,20 @@ async fn main() {
             println!("YubiKey config:");
             println!("{cfg}")
         }
+
+        #[cfg(feature = "usb")]
+        Opt::Wink => {
+            use webauthn_authenticator_rs::usb::USBTransport;
+
+            println!("Insert a USB device...");
+            let transport = USBTransport::new().await.unwrap();
+            let mut stream = transport.watch().await.unwrap();
+            while let Some(event) = stream.next().await {
```

Comment:
This will just wink forever until it fails, and doesn't tell you when winks succeeded. A bunch of authenticators will normally flash when you plug them in, so it's hard to tell from the software side if it's doing anything. 😉 

I think this should work a bit more like `info`: by default it only winks the first authenticator which supports blinking, but have a flag which makes it wink forever. And maybe tell you explicitly if a connected device *doesn't* support winking.

Reply:
Ack, makes sense, thanks for the feedback.
I've pushed an additional commit on top which I think takes care of all your suggestions.